assets/js/klarna-checkout-for-woocommerce.js

```diff
@@ -231,7 +231,17 @@ jQuery( function ( $ ) {
 						if ( name === "shipping_email" ) {
 							kco_wc.shippingEmailExists = true
 						}
-						$( "p#" + name + "_field" ).appendTo( "#kco-extra-checkout-fields" )
+
+						$( ".woocommerce-account-fields" ).appendTo( "#kco-extra-checkout-fields" )
+
+						const alreadyMoved = [
+							"account_username",
+							"account_password",
+						];
+
+						if( ! alreadyMoved.includes( name ) ) {
+							$( "p#" + name + "_field" ).appendTo( "#kco-extra-checkout-fields" )
```

Comment:
This logic is very similar to how we already filter out some fields. Could we not add the account_username and account_password field names to the property `standard_woo_checkout_fields` when enqueueing the file [here](https://github.com/krokedil/klarna-checkout-for-woocommerce/blob/6bbae5272f2d661e80702379dee030df0fb455b4/classes/class-kco-gateway.php#L377) instead to keep the logic the same?